src/app/components/product/product.harness.ts

```diff
@@ -0,0 +1,29 @@
+import { ComponentHarness } from '@angular/cdk/testing';
+
+export class ProductHarness extends ComponentHarness {
+  static hostSelector = 'tk-product';
+
+  getTitle = this.locatorFor('mat-card-title');
+  getImage = this.locatorFor('[mat-card-image]');
+  getDescription = this.locatorFor('mat-card-content p');
+
+  async titleText(): Promise<string> {
+    const title = await this.getTitle();
+    return title.text();
+  }
+
+  async imgSrc(): Promise<string> {
```

Comment:
Merge those two methods and return the image object itself.

Reply:
Ok, thats logical.
But I read that returning `TestElement` is bad practice and we should avoid it. Thats why I created 2 separate methods, or that's acceptable practice in Google ?